<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerView</class>
 <widget class="QDialog" name="CustomerView">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>顾客</string>
  </property>
  <widget class="QGroupBox" name="gbSeatPick">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>275</y>
     <width>550</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>175</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblHint">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbMenu">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>850</width>
     <height>650</height>
    </rect>
   </property>
   <property name="title">
    <string>查看菜单并点餐</string>
   </property>
   <widget class="QTableWidget" name="tbMenuDisp">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>575</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmit">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>575</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>提交订单并付款</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>575</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>返回重新选桌</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbOrder">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>550</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>查看订单完成情况</string>
   </property>
   <widget class="QTableWidget" name="tbOrder">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbGeneral">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFinish">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>440</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>用餐完毕</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgress">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>完成进度：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbInstruction">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>175</y>
     <width>350</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string>额外命令</string>
   </property>
   <widget class="QLabel" name="lblWaiter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddWater">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>加水</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPackUp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>打包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHasten">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>催单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComDish">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>400</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>评价已点菜品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComWaiter">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>450</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>评价服务员</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnProfile">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>175</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>个人资料</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCustomerBGN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>1100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblCustomerBGN</zorder>
  <zorder>gbSeatPick</zorder>
  <zorder>lblInfo</zorder>
  <zorder>lblHint</zorder>
  <zorder>gbMenu</zorder>
  <zorder>gbOrder</zorder>
  <zorder>gbInstruction</zorder>
  <zorder>btnProfile</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>PhaseSeatPick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSubmit</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>SubmitOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddWater</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnAddWater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPackUp</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnPackUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>882</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHasten</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnHasten()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFinish</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnFinish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnComDish</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnCommentDish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnComWaiter</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnCommentWaiter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>800</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>926</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProfile</sender>
   <signal>clicked()</signal>
   <receiver>CustomerView</receiver>
   <slot>OnProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PhaseSeatPick()</slot>
  <slot>SubmitOrder()</slot>
  <slot>OnAddWater()</slot>
  <slot>OnPackUp()</slot>
  <slot>OnHasten()</slot>
  <slot>OnCommentDish()</slot>
  <slot>OnFinish()</slot>
  <slot>OnCommentCook()</slot>
  <slot>OnCommentWaiter()</slot>
  <slot>OnProfile()</slot>
 </slots>
</ui>
